stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1230,7 +1230,7 @@ function det(A::AbstractMatrix{T}) where T
         S = typeof((one(T)*zero(T) + zero(T))/one(T))
         return convert(S, det(UpperTriangular(A)))
     end
-    return det(lu(A))
+    return det(lu(A; check = false))
```

Comment:
Does this `check = false` belong here?

Reply:
Yes, otherwise we throw, and `det` deals with failed factorizations:
https://github.com/JuliaLang/julia/blob/c8ce43ad173a9126e902dda246b74cb6e83eaded/stdlib/LinearAlgebra/src/lu.jl#L333